luxstat.tex

```diff
@@ -204,15 +198,16 @@ \subsection{Mitgliederversammlung}
 \subsection{Vorstand}
 
 \luxarticle
-Der Vorstand besteht aus mindestens vier gewählten 
+Der Vorstand besteht aus mindestens fünf gewählten 
 Aktivmitgliedern.
 
 Er besteht namentlich aus:
 \begin{itemize}
-\item Präsident
-\item Vizepräsident
-\item Kassier
-\item Aktuar
+\item Präsident:in
+\item Vizepräsident:in
+\item Kassier:in
+\item Aktuar:in
+\item Infrastrukturverantwortlicher:in
```

Comment:
```suggestion
\item Infrastrukturverantwortliche:r
```

Allerdings wäre ich eh dafür, die/den Infrastrukturverantwortliche/n nicht in die Statuten aufzunehmnen (siehe oben), da rechtlich nicht relevant.